### ISSUE TITLE
Reject SPF & bounces logs in webadmin

### ISSUE THREAD

**Issue by ravivgolov** [zone-eu/zone-mta#385]
Hi

Is there any way to reject a message when the domain does not have the hostname SPF record?
Similiar to Haraka?
https://haraka.github.io/plugins/spf

Another issue, I do not need to see the bounces list? or delivery?
<img width="1202" alt="Screenshot 2024-04-22 110304" src="https://github.com/zone-eu/zone-mta/assets/99405975/437a4512-4697-4a58-af37-06aab4242396">
<img width="959" alt="Screenshot 2024-04-22 110531" src="https://github.com/zone-eu/zone-mta/assets/99405975/23615ee2-5531-429c-9900-b8d02411e971">

**Comment by louis-lau** [comment]
Not aware of any plugins that do this so far, but it should be fairly easy to create. Just like haraka, zonemta has plugin capability. Be aware that forwarding DKIM signed email passes dmarc but fails SPF. So if you process that type of emails the check will fail for legitimate emails.

> Another issue, I do not need to see the bounces list? or delivery?

What do you mean by this question? Please elaborate.

**Comment by ravivgolov** [comment]
What list of emails I should see in the web admin?

Where I can see the list of all outbound emails including bounces/delivered.

**Comment by louis-lau** [comment]
You'd very rarely need such a list in a production system with a high volume of email. I guess this is why it isn't a feature in the web admin interface.

You can try searching using the date though, like this: `start:2018-04-10T09:14:46Z`

Will show all messages sent after that date (which will probably just be all messages in your case)

**Comment by ravivgolov** [comment]
I see, I came from Haraka to test if this one is better,
What is the best way to monitor logs? I want to monitor spam, phishing, and email marketing, but I don’t see any built-in plugins that assist with this.

By the way, how can I enable the Rspamd plugin, which is already installed on my system, in the template folder?

Lastly, I was able to send an email using a fake address like [example@amazon.com](mailto:example@gmail.com), and it was delivered to my Gmail inbox. This is why I want to enable SPF checks, to reject phishing.
